user-guide/Advanced_Functionality/DataMiner_Agents/Installing_a_DMA/Installing_DM_using_the_DM_Installer.md

```diff
@@ -0,0 +1,111 @@
+---
+uid: Installing_DM_using_the_DM_installer
+---
+
+# Installing DataMiner using the DataMiner Installer
+
+## Prerequisites
+
+Before you run the installer, install the latest .NET Framework version.
+
+- For more information on recommended versions, see [DataMiner Compute Requirements](https://community.dataminer.services/dataminer-compute-requirements/).
+- For installation information, see <https://docs.microsoft.com/en-us/dotnet/framework/install/>
+
+In addition, make sure the Windows setting "fast startup" is not activated.
+
+## Running the installer
+
+The DataMiner installer allows you to run a default DataMiner installation, which includes a Cassandra database on the C drive, or to run a custom installation. A custom installation can for instance be used to install a MySQL database instead of a Cassandra database.
+
+> [!NOTE]
+> The default installation requires that the built-in Windows Administrator account is enabled and that WinPcap is installed.
+
+### Default DataMiner installation
+
+1. Make sure you are logged into Windows with the Administrator account. (Do not use a regular user account with administrative rights.)
+
+1. Double-click *Setup.exe*.
+
+1. Click *Install*.
+
+1. Enter the DataMiner ID.
+
+   > [!IMPORTANT]
+   > To get this DataMiner ID, you must contact Skyline. The DataMiner ID will uniquely identify the DataMiner Agent you are installing.
+
+1. Click next.
+
+   The progress of the installation will be displayed. A *cancel* button in the lower right corner allows you to cancel the installation process if necessary.
+
+1. Once the installation is complete, click *next*.
+
+1. Click *go to Request.lic* to browse to *Request.lic*.
+
+1. Send the *Request.lic* file to <dataminer.licensing@skyline.be>, and wait until you receive the reply.
+
+1. Once you have received the license files, save these somewhere on the computer; however, not in the "Skyline DataMiner" folder.
```

Comment:
You can also copy the .lic files to C:\Skyline DataMiner, should we add this as an alternative step (sometimes people close the installer before the licensing has been uploaded)?